docs/backends/vllm/multimodal_vllm_guide.md

```diff
@@ -0,0 +1,204 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# vLLM Multimodal Guide
+
+This document provides a comprehensive guide for multimodal inference using vLLM backend in Dynamo.
+
+## Multimodal Support Matrix
+
+| Modality | Input Format | Aggregated | Disaggregated | Notes |
+|----------|--------------|------------|---------------|-------|
+| **Image** | HTTP/HTTPS URL | Yes | Yes | Full support for all image models |
+| **Image** | Data URL (Base64) | Yes | Yes | Inline base64-encoded images |
+| **Video** | HTTP/HTTPS URL | Yes | Yes | Frame extraction and processing |
+| **Audio** | HTTP/HTTPS URL | Yes | Yes | Experimental - requires audio dependencies |
+
+## Architecture Comparison
+
+vLLM multimodal supports three deployment patterns:
+
+```
+SIMPLE AGGREGATED (agg_multimodal.sh):
+  Client → Frontend (Rust) → Worker [image load, encode, P+D] → Response
+  • 2 components • --connector none • Easiest setup
+
+EPD AGGREGATED (agg_multimodal_epd.sh):
+  Client → Frontend → Processor → Encoder [NIXL] → PD Worker → Response
+  • 4 components • --multimodal-processor • Custom templates, NIXL
+
+DISAGGREGATED (disagg_multimodal_qwen.sh):
+  Client → Frontend → Processor → Encoder [NIXL] → Prefill [NIXL] → Decode → Response
+  • 5 components • Separate P/D workers • Multi-node, max optimization
+```
+
+## Input Format Details
+
+### Supported URL Formats
+
+| Format | Example | Description | Support |
+|--------|---------|-------------|---------|
+| **HTTP/HTTPS** | `http://example.com/image.jpg` | Remote media files | ✅ |
+| **Data URL** | `data:image/jpeg;base64,/9j/4AAQ...` | Base64-encoded inline data | ✅ |
+
+
+## Aggregated Mode (PD)
```

Comment:
Could we clarify here it's the EPD not the simple aggregated one? Do we want to add a section for the simple aggregated workflow? I think one confusion that I got frequently from people is EPD vs simple/traditional. If we could align on the wording that would be very helpful.